src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -29,10 +29,10 @@ Supported Platforms:
   <files>
     <file src="Microsoft.CodeAnalysis.Workspaces.dll" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.Workspaces.xml" target="lib\portable-net45+win8" />
```

Comment:
@jasonmalinowski - On testing this change, I am noticing the following behavior.

1. If I create a console app targeting netfx 4.5.2 (the default) and install the MS.CA.Workspaces.Common package containing the below change, looks like nuget adds the above `portable-net45+win8` reference to MS.CA.Workspaces.dll and there is no reference to MS.CA.Workspaces.Desktop.

2. On the other hand, if I create a console app targeting netfx 4.6 and install the same MS.CA.Workspaces.Common package, nuget adds reference to the below two `lib\net46` dlls (i.e. both MS.CA.Workspaces.dll and MS.CA.Workspaces.Desktop.dll).

Is it a bug that nuget allows the portable reference to be added in 1 above (even though the console app is not portable)?

If not, is this the behavior we want for this package (i.e. should we bump `lib\portable-net45+win8` above to `lib\portable-46+win8` to avoid this discrepancy and make this package not installable on projects targeting netfx 4.5.2)?